src/dapps/selectors.ts

```diff
@@ -117,6 +117,46 @@ export const favoriteDappsSelector = createSelector(
   }
 )
 
+export const favoriteDappsWithCategoryNamesSelector = createSelector(
```

Comment:
nit: These 2 functions (`favoriteDappsWithCategoryNamesSelector` and  `dappListWithCategoryNamesSelector`) are pretty similar, can you put the code in an auxiliar function to avoid the duplicated code?